.env.example

```
@@ -1,5 +1,6 @@
 PORT=
 DISCORD_TOKEN=
+RUN_DISCORD=
```

```suggestion
ENABLE_DISCORD_BOT=
```

This might be slightly more clear